lib/internal/test_runner/mock/fake_timers.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+const {
+  DateNow,
+  globalThis,
+} = primordials;
+
+const PriorityQueue = require('internal/priority_queue');
+
+function compareTimersLists(a, b) {
+  return (a.runAt - b.runAt) || (a.id - b.id);
+}
+
+function setPosition(node, pos) {
+  node.priorityQueuePosition = pos;
+}
+class Timers {
+  #currentTimer = 1;
+  constructor() {
+    this.timers = new PriorityQueue(compareTimersLists, setPosition);
```

Comment:
```suggestion
 #timers
  constructor() {
    this.#timers = new PriorityQueue(compareTimersLists, setPosition);
```
probably a lot can be private fields

Reply:
https://github.com/nodejs/node/pull/47775#discussion_r1181860407

Reply:
fixed!